cpu/pic32/Makefile.pic32

```diff
@@ -43,6 +46,13 @@ endif
 
 ### Compilation rules
 
+CUSTOM_RULE_LINK=1
+
+%.$(TARGET): %.co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a
```

Comment:
even though its specific to PIC32, it will come into picture when we are building a project, besides this is how it's done on creator branch